src/pretix/base/models/discount.py

```diff
@@ -0,0 +1,355 @@
+    SUBEVENT_MODE_DISTINCT = 'distinct'
+    SUBEVENT_MODE_CHOICES = (
+        (SUBEVENT_MODE_MIXED, pgettext_lazy('subevent', 'Dates can be mixed without limitation')),
+        (SUBEVENT_MODE_SAME, pgettext_lazy('subevent', 'All matching products must be for the same date')),
+        (SUBEVENT_MODE_DISTINCT, pgettext_lazy('subevent', 'Each matching product must be for a different date')),
+    )
+
+    event = models.ForeignKey(
+        'Event',
+        on_delete=models.CASCADE,
+        related_name='discounts',
+    )
+    active = models.BooleanField(
+        verbose_name=_("Active"),
+        default=True,
+    )
+    internal_name = models.CharField(
+        verbose_name=_("Internal name"),
+        max_length=255
+    )
+    position = models.PositiveIntegerField(
+        default=0,
+        verbose_name=_("Position")
+    )
+    sales_channels = fields.MultiStringField(
+        verbose_name=_('Sales channels'),
+        default=['web'],
+        blank=False,
+    )
+
+    available_from = models.DateTimeField(
+        verbose_name=_("Available from"),
+        null=True,
+        blank=True,
+    )
+    available_until = models.DateTimeField(
+        verbose_name=_("Available until"),
+        null=True,
+        blank=True,
+    )
+
+    subevent_mode = models.CharField(
+        verbose_name=_('Event series handling'),
+        max_length=50,
+        default=SUBEVENT_MODE_MIXED,
+        choices=SUBEVENT_MODE_CHOICES,
+    )
+
+    condition_all_products = models.BooleanField(
+        default=True,
+        verbose_name=_("Apply to all products (including newly created ones)")
+    )
+    condition_limit_products = models.ManyToManyField(
+        'Item',
+        verbose_name=_("Apply to specific products"),
+        blank=True
+    )
+    condition_apply_to_addons = models.BooleanField(
+        default=True,
+        verbose_name=_("Apply to add-on products"),
+        help_text=_("Discounts never apply to bundled products"),
+    )
+    condition_min_count = models.PositiveIntegerField(
+        verbose_name=_('Minimum number of matching products'),
+        default=0,
+    )
+    condition_min_value = models.DecimalField(
+        verbose_name=_('Minimum gross value of matching products'),
+        decimal_places=2,
+        max_digits=10,
+        default=Decimal('0.00'),
+    )
+
+    benefit_discount_matching_percent = models.DecimalField(
+        verbose_name=_('Percentual discount on matching products'),
+        decimal_places=2,
+        max_digits=10,
+        default=Decimal('0.00'),
+        validators=[MinValueValidator(Decimal('0.00'))],
+    )
+    benefit_only_apply_to_cheapest_n_matches = models.PositiveIntegerField(
+        verbose_name=_('Apply discount only to this number of matching products'),
+        help_text=_('Keep the field empty to apply the discount to all matching products.'),
+        null=True,
+        blank=True,
+        validators=[MinValueValidator(1)],
+    )
```

Comment:
To me this reads more like a Jeopardy clue ;-)

I think it needs more explanation along the lines of "Setting this to two will apply the discount only to groups of 2 tickets. Buying three tickets would grant two tickets with discount, one at full price. Four tickets though would give 4 discounted tickets".

Reply:
I don't think your example is quite perfect yet, "Setting this to two will apply the discount only to groups of 2 tickets" is confusing, because with condition_min_count=3 and benefit_only_apply_to_cheapest_n_matches=2, it will *look* for groups of *three*. However, I pushed a new text inspired by it, do you think it's better?

Reply:
:+1: